rust/toolchain.bzl

```diff
@@ -535,6 +537,16 @@ rust_toolchain = rule(
             ),
             mandatory = True,
         ),
+        "llvm_cov": attr.label(
```

Comment:
There seems to be a subtlety here in that these llvm tools for rust and from the underlying C++ toolchain may not necessarily be compatible with each other, especially the one working with the raw profiles: https://clang.llvm.org/docs/SourceBasedCodeCoverage.html#format-compatibility-guarantees
So it seems at least having the ability to set these tools explicitly is good.

Reply:
Yea 😕, these versions could be very different, on macOS at the moment Xcode's built in tools haven't been updated to LLVM 14, where rust's have, so you have to use rust's instead. I assume that problem will cause other issues later as well, but this is probably better otherwise. Also if you were using GCC I guess we'd have to use rust's versions as well.